src/server.ts

```
@@ -0,0 +1,381 @@
+              ],
+            };
+          }
+          
+          case 'run_orchestrator':
+            const result = await this.orchestrator.run(
+              args?.repo as string,
+              args?.prNumber as number,
+              args?.maxIterations as number,
+              args?.dryRun as boolean,
+              args?.validationMode as 'internal' | 'external'
+            );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)